app/models/host.rb

```diff
@@ -1407,7 +1405,7 @@ def scan_from_queue(taskid = nil)
 
             save
           end
-        rescue MiqException::MiqSshUtilHostKeyMismatch
+        rescue Net::SSH::HostKeyMismatch
```

Comment:
cc @himdel since this is used in the UI

Reply:
ah, the old name is gone?

In that case, https://github.com/ManageIQ/manageiq-ui-classic/pull/6686 :)

Reply:
Does anything need to be handled on the UI side similarly for the above code in ConversionHost?  cc @fdupont-redhat

Reply:
Nope. We don't check credentials V2V plugin.